app/src/main/java/com/imdbmovieapp/data/remote/dto/searchMoviesDto/SearchMoviesDto.kt

```diff
@@ -0,0 +1,8 @@
+package com.imdbmovieapp.data.remote.dto.searchMoviesDto
+
+data class SearchMoviesDto(
+    val page: Int,
+    val results: List<Any>,
+    val total_pages: Int,
+    val total_results: Int
```

Comment:
We don't name variables with "_". Name it like totalPages, totalResults (fix names in other classes as well)